plugins/usertiming.js

```diff
@@ -87,21 +93,49 @@
 		options: {"from": 0, "window": BOOMR.window},
 
 		/*
+		 * Gets the user timings, filters out those that have already been sent
 		 * Calls the UserTimingCompression library to get the compressed UserTiming
-		 * data that occurred since the last call.
+		 * data that occurred since the last call
 		 *
-		 * @returns {string} Compressed UserTiming data
+		 * @returns {string} UserTiming data
 		 */
 		getUserTiming: function() {
-			var timings, res,
-			    now = BOOMR.hrNow(),  // Get a timestamp to compare with performance marks and measures.
-			    utc = window.UserTimingCompression || BOOMR.window.UserTimingCompression;
+			var frame, entries, options;
 
-			timings = utc.getCompressedUserTiming(impl.options);
-			res = utc.compressForUri(timings);
-			this.options.from = now;
+			options = impl.options || {};
+			frame = options.window || window;
+			entries = this.findUserTimingForFrame(frame);
```

Comment:
Then this would be just `entries = this.findUserTimingForFrame(options.window);`